components/doc/common/codeeditor/index.js

```diff
@@ -1,13 +1,20 @@
 import sdk from '@stackblitz/sdk';
-import { getCRA, getNextJS } from './templates';
+import { getVite } from './templates';
 
 const useCodeSandbox = (props) => {
     const getSandboxParameters = (sourceType) => {
-        const { files, dependencies, sourceFileName } = props.template === 'cra' ? getCRA(props, sourceType) : getNextJS(props, sourceType);
+        const sandboxDependencies = {
+            vite: '^4.0.0',
+            '@vitejs/plugin-react': '^4.0.0'
+        };
+        const { files, dependencies, sourceFileName } = getVite({ dependencies: sandboxDependencies, ...props }, sourceType);
 
         files['sandbox.config.json'] = {
             content: {
-                infiniteLoopProtection: false
+                infiniteLoopProtection: false,
+                container: {
+                    node: '16'
```

Comment:
Should we use Node 18 or 20 since those are LTS now?

Reply:
Codesandbox not supporting 18 or 20. Unfortunately this is the lates version that we can use. This is why we are also using the vite 4.0.0 for codesandbox

Reply:
When i set the node version higher than 16 it sets the node version to the default value which is 10


![image](https://github.com/primefaces/primereact/assets/29578451/2ddcdcdb-89cd-437b-a18d-b6e5e61d4eae)

Reply:
Codesandbox have this buckets called DevBox and it supports Node 18 or 20 we can basically use whatever we want but I couldn't find any examples about creating DevBox buckets via API

Reply:
OK no problem thanks for checking!